homeassistant/components/plugwise/entity.py

```diff
@@ -60,6 +60,20 @@ def __init__(
                 }
             )
 
+    async def async_send_api_call(self, target: str, command: str) -> bool:
```

Comment:
This can be put into the entity descriptions directly, as in, on the place used. There is no need to put this to the base entity.

Reply:
Ok, thanks, I will look at this during tomorrow. Time to relax before it's time to sleep ;)

Reply:
Updates as suggested.